pkg/svc/extension.go

```diff
@@ -361,41 +366,43 @@ func (s *extensionRegistrySvc) ListServices(ctx context.Context, req *extension.
 func (s *extensionRegistrySvc) UpdateService(ctx context.Context, req *extension.ExtensionService) (res *extension.ExtensionService, err error) {
 	s.logger.Print("extension.updateService")
 	service := extensionServiceToDomain(req)
-	svcRecord, err := s.registry.GetService(ctx, domain.ExtensionServiceID{
-		ID:          service.ID,
-		ExtensionID: service.ExtensionID,
-	}, false)
+	svc, err := s.registry.GetService(ctx, *req.ExtensionID, service.ID)
```

Comment:
same thing here: what if `req.ExtensionID` is nil ?

Reply:
Fixed by using `util.DerefString`